packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -170,6 +170,7 @@ class DatePickerInput extends React.Component {
           placeholder={placeholder}
           onFocus={onFocus}
           onBlur={onBlur}
+          aria-label={this.context.intl.formatMessage({ id: 'Terra.datePicker.date' })}
```

Comment:
This will have to be changed to use react-intl's context once after #768 is merged to the master.

Reply:
@neilpfeiffer . Please check the aria label value added and acknowledge if I can go ahead and log a JIRA for translations.

Reply:
Talked about it in our war room today and this looks good from our side.  Feel free to log the JIRA translations

Reply:
Translations Added in a6b22a4